lib/src/arithmetic/signals/floating_point_logics/floating_point_logic.dart

```diff
@@ -73,13 +77,17 @@ class FloatingPoint extends LogicStructure {
   FloatingPointValuePopulator valuePopulator() => FloatingPointValue.populator(
       exponentWidth: exponent.width,
       mantissaWidth: mantissa.width,
-      explicitJBit: explicitJBit);
+      explicitJBit: explicitJBit,
+      subNormalAsZero: subNormalAsZero);
 
   /// Return true if the J-bit is explicitly represented in the mantissa.
   bool get explicitJBit => _explicitJBit;
-
   late final bool _explicitJBit;
 
+  /// Return true if subnormal numbers are represented as zero.
+  bool get subNormalAsZero => _subNormalAsZero;
```

Comment:
Should we add a simple converter function to the `FloatingPointLogic` so that you can convert to that type?  E.g. imagine you have a "normal" FP Logic and then want to pass it into an FP Adder with SAZ on the inputs?  It seems like you could just say `toSubNormalAsZero` and it would create the `mux` and output the proper type on the resulting conversion post-mux?  This could mean the muxing becomes redundant inside an adder (or other component), but synthesis could take care of it?

Also/instead, should we expand the `clone` function to take an argument to convert it to that type so that you can easily pass in an output with everything the same (widths, etc.) except just changing the SAZ?